### PR TITLE
Add customer offers page

### PR DESCRIPTION
**Pull request by Ken-Lo**
Do Not Merge

- [x] Confirm copy with partners
- [x] Add in Boost Capital landing page URL

### CODE REVIEW

**Comment by Ken-Lo** [comment]
Hey @jamesshedden 
We still need to confirm copy with our partners, but would you mind reviewing the code in the mean time?
Thanks v much! :)

File: app/pages/customer-offers/customer-offers.js [hunk 1]
```diff
@@ -0,0 +1,87 @@
+import React from 'react';
+import Page from '../../components/page/page';
+
+// For each merchant offer, need to set up a relevant redirect to the partner's landing page in config/redirects.json
```
Comment (jamesshedden) [inline app/pages/customer-offers/customer-offers.js]: Do we need to keep this comment? ✂️

**Comment by jamesshedden** [comment]
@Ken-Lo Hey, nice work, code looks good — did you work with a designer on this? If so be good to ping them in this thread also for a 'Design Review' before we ship anything — cheers! ⚡️

**Comment by Ken-Lo** [comment]
Thanks @jamesshedden - much appreciated! This is a coded-up version of Josh's design - I'll catch up with him tomorrow before we merge in.

**Comment by jamesshedden** [comment]
@Ken-Lo Sounds good, yeah be great to loop him in on this — nice one 🌵

**Pull request merged**